clang/lib/CodeGen/CodeGenModule.h

```diff
@@ -587,6 +587,10 @@ class CodeGenModule : public CodeGenTypeCache {
   /// A vector of metadata strings for dependent libraries for ELF.
   SmallVector<llvm::MDNode *, 16> ELFDependentLibraries;
 
+  /// Single module-level copyright comment (if any).
```

Comment:
Avoid "module" without qualification. "Single" is also redundant given the next sentence.
```suggestion
  /// Metadata for copyright pragma comment (if present).
```